Makefile

```diff
@@ -112,6 +112,7 @@ LIVE=$(BUILD_DIR)/live
 LIVE_IMG=$(BUILD_DIR)/live.$(IMG_FORMAT)
 TARGET_IMG=$(BUILD_DIR)/target.img
 INSTALLER=$(BUILD_DIR)/installer
+INSTALLER_ROOTFS_YML=$(INSTALLER)/linuxkit-config-version.yaml
```

Comment:
We store `linuxkit-eve-config.yml` in rootfs, but `linuxkit-config-version.yaml` here. Is there any reason to use different names?

Reply:
yes, a very good reason. Copy-paste error. 😆

Reply:
Fixed. Thanks @giggsoff